lib/client.js

```diff
@@ -1,7 +1,12 @@
-Kadira.profileCpu = function(arg1, arg2, type) {
+Kadira.profileCpu = function (arg1, arg2, type) {
+  if (!this.connected) {
+    return console.error("You need to have the montiapm:agent connected");
+  }
```

Comment:
This package can be used for both local and remote profiles, and only remote profiles require the agent to be connected (local profiles are saved locally on disk so it doesn't need to communicate with Monti APM).

When a profile is remote, `type` is set to `remote`.